### PR TITLE
SEO-3260-ASP.NET-Missing-H1

### PR DESCRIPTION
**Pull request by berylouma**
Hi @Aishwarya-Ganesan 

I have worked on missing H1

Regards,
Beryl

### CODE REVIEW

File: aspnet/Gantt/How-to/Change-Weekend.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
 ---
 layout: post
-title: Change-Weekend | Gantt | ASP.NET Webforms | Syncfusion
-description: Change workweek/working days in a week
+title: Change Weekend | Gantt | ASP.NET Webforms | Syncfusion
+description: Learn here about how to change workweek/working days in Syncfusion Essential ASP.NET Gantt Control, its elements, and more.
```
Comment (gowthamanthavasiyappan) [inline aspnet/Gantt/How-to/Change-Weekend.md:4]: Change ASP.NET WebForms

File: aspnet/Gantt/How-to/Change-Weekend.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 platform: aspnet
 control: Gantt
 documentation: ug
 ---
 
-## Change workweek/working days in a week
+# Change workweek/Weekend in ASP.NET Gantt
```
Comment (gowthamanthavasiyappan) [inline aspnet/Gantt/How-to/Change-Weekend.md:10]: Change like this - Change workweek/Weekend in ASP.NET WebForms Gantt

File: aspnet/Grid/How-to/Apply-formatting-for-columns-dynamically.md [hunk 3]
```diff
@@ -1,13 +1,13 @@
 ---
 layout: post
-title: Apply formatting for columns dynamically | Grid | ASP.NET Webforms | Syncfusion
-description: apply formatting for columns dynamically
+title: Apply formatting for columns | Grid | ASP.NET Webforms | Syncfusion
+description: Learn here about how to apply formatting for columns dynamically in Syncfusion Essential ASP.NET Grid Control, its elements, and more.
```
Comment (gowthamanthavasiyappan) [inline aspnet/Grid/How-to/Apply-formatting-for-columns-dynamically.md:4]: Change ASP.NET WebForms

File: aspnet/Grid/How-to/Apply-formatting-for-columns-dynamically.md [hunk 4]
```diff
@@ -1,13 +1,13 @@
...
 platform: aspnet
 control: Grid
 documentation: ug
 ---
 
-## Apply formatting for columns dynamically
+# Apply formatting for columns dynamically in ASP.NET Grid
```
Comment (gowthamanthavasiyappan) [inline aspnet/Grid/How-to/Apply-formatting-for-columns-dynamically.md:10]: Change ASP.NET WebForms

File: aspnet/Grid/How-to/Initialize-Grid.md [hunk 5]
```diff
@@ -1,13 +1,13 @@
 ---
 layout: post
 title: Initialize Grid | Grid | ASP.NET Webforms | Syncfusion
-description: initialize grid
+description: Learn here about how to initialize grid in Syncfusion Essential ASP.NET Grid Control, its elements, and more.
```
Comment (gowthamanthavasiyappan) [inline aspnet/Grid/How-to/Initialize-Grid.md:4]: Change ASP.NET WebForms

**Review by gowthamanthavasiyappan — CHANGES_REQUESTED** [review]

File: aspnet/Gantt/How-to/Change-Weekend.md (hunk 2 above)
Comment (berylouma) [reply aspnet/Gantt/How-to/Change-Weekend.md:10]: Done

File: aspnet/Grid/How-to/Apply-formatting-for-columns-dynamically.md (hunk 3 above)
Comment (berylouma) [reply aspnet/Grid/How-to/Apply-formatting-for-columns-dynamically.md:4]: Done

File: aspnet/Grid/How-to/Apply-formatting-for-columns-dynamically.md (hunk 4 above)
Comment (berylouma) [reply aspnet/Grid/How-to/Apply-formatting-for-columns-dynamically.md:10]: Done

File: aspnet/Grid/How-to/Initialize-Grid.md (hunk 5 above)
Comment (berylouma) [reply aspnet/Grid/How-to/Initialize-Grid.md]: Done

**Pull request closed without merging**